torch/onnx/symbolic_opset12.py

```diff
@@ -13,3 +13,47 @@
 def einsum(g, equation, tensor_list):
     tensors = sym_help._unpack_list(tensor_list)
     return g.op("Einsum", *tensors, equation_s=equation)
+
```

Comment:
do we need to put parse_args here?

Reply:
no since _maybe_get_const is used on the non-tensor inputs